plugin-modernizer-core/src/main/java/io/jenkins/tools/pluginmodernizer/core/utils/TemplateUtils.java

```diff
@@ -46,22 +57,25 @@ public static String renderCommitMessage(Plugin plugin, Recipe recipe) {
 
     /**
      * Render the pull request title
+     *
      * @param plugin Plugin to modernize
      * @param recipe Recipe to apply
      * @return The rendered pull request title
      */
     public static String renderPullRequestTitle(Plugin plugin, Recipe recipe) {
         if (hasTitleTemplate(recipe)) {
-            String shortName = recipe.getName().replaceAll(Settings.RECIPE_FQDN_PREFIX + ".", "");
-            return renderTemplate("pr-title-%s.jte".formatted(shortName), Map.of("plugin", plugin, "recipe", recipe));
+            return renderTemplate(
```

Comment:
I think it create some regression ?

Error rendering template pr-title-pr-title-UpgradeBomVersion.jte.jte

The double 'jte' looks suspicious